tests/e2e/pubsub/jetstream/authentication/authentication_test.go

```diff
@@ -53,6 +54,11 @@ func TestAuthentication(t *testing.T) {
 			}
 			err := js.Init(md)
 			if err != nil && !tC.expectError {
+				if strings.Contains(err.Error(), "network is unreachable") ||
+					strings.Contains(err.Error(), "no servers available for connection") {
```

Comment:
Why it can not start as expected?

Reply:
When using `go test -v ./...`, jetstream is not actually started locally. At present, if this unit test is to pass, the jetstream server must be started locally through ` docker-compose.yaml`.

Reply:
Another problem:

I passed the unit tests last week, but after merging the community code yesterday, I found that the unit tests could not run, and there were the following errors:

<img width="1199" alt="image" src="https://user-images.githubusercontent.com/20457624/183796684-c9889f34-69de-4d6b-81ab-673df4463feb.png">

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/20457624/183796839-3bc5b2c8-55ae-40ed-a874-9b5a65fe716f.png">